code/modules/awaymissions/corpse.dm

```diff
@@ -40,6 +40,30 @@
 	M.timeofdeath = timeofdeath
 	if(src.mob_species)
 		M.set_species(src.mob_species)
+	if(ishuman(M))
+//		random_character()
+//		M.prefs.random_character()

```

Comment:
Sorry. I was trying stuff and needed to change computers, so uploaded latest everything.